definitions/tools/docm_add_variants.cwl

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "CombineVariants (GATK 3.6)"
+baseCommand: ["/usr/bin/java", "-Xmx8g", "-jar", "/opt/GenomeAnalysisTK.jar", "-T", "CombineVariants"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 9000
+      tmpdirMin: 25000
+    - class: DockerRequirement
+      dockerPull: mgibio/cle
+arguments:
+    ["-genotypeMergeOptions", "PRIORITIZE",
+     "--rod_priority_list", "callers,docm",
+     "--setKey","null",
```

Comment:
This is the second combine_variant set. So I assume those "set=<real_variant_callers>" in INFO field from the first combine_variant step will be preserved. But "null" here will not add "docm" to "set=". So what if docm variants is also called by real_callers ? There won't be something like "set=<real_caller>,docm". If the variant is only called by docm caller, it can probably be identified by INFO field without "set=".

Reply:
It's a good point, but if a docm variant is called by a real caller, I can't think of a reason why we'd need to know that.  The point of docm is to "rescue" things that weren't called by the standard callers, in my mind.  Does this clash with some of the existing CLE downstream workflow?